.github/workflows/RFC-2119-usage.yml

```diff
@@ -0,0 +1,16 @@
+name: "Check if PR introduces errant RFC-2119"
+on: 
+  pull_request:
+      types: [opened, edited, assigned, closed, review_requested, ready_for_review]
```

Comment:
I'm a little confused about this list, I'm not super familiar with github workflows, but does this check need to be run in all these scenarios? shouldn't it only run if the PR is updated/edited? Or am I'm miss understanding this list..

Reply:
agree! i updated it just to include `opened/edited`